sensor/kubernetes/clustermetrics/cluster_metrics.go

```diff
@@ -12,17 +13,26 @@ import (
 	"github.com/stackrox/rox/sensor/common"
 	"github.com/stackrox/rox/sensor/common/message"
 	metricsPkg "github.com/stackrox/rox/sensor/common/metrics"
+	"github.com/stackrox/rox/sensor/kubernetes/complianceoperator"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 )
 
-var log = logging.LoggerForModule()
-
-// Interval for querying cluster metrics from Kubernetes and sending to Central.
-var defaultInterval = 5 * time.Minute
+var (
+	log = logging.LoggerForModule()
+	// Interval for querying cluster metrics from Kubernetes and sending to Central.
+	defaultInterval = 5 * time.Minute
+	// Timeout for querying cluster metrics from Kubernetes.
+	defaultTimeout = 10 * time.Second
+)
 
-// Timeout for querying cluster metrics from Kubernetes.
-var defaultTimeout = 10 * time.Second
+const (
+	// coAvailableInUnknownVersion is value used in telemetry
+	// when we find Compliance Operator but are unable to extract its version
+	coAvailableInUnknownVersion = "CO available in unknown version"
+	// coUnavailable is value used in telemetry when we do not find Compliance Operator deployment
+	coUnavailable = "CO unavailable"
```

Comment:
The property name is "Compliance Operator Version", so the values could be simply
```suggestion
	coAvailableInUnknownVersion = "unknown"
	// coUnavailable is value used in telemetry when we do not find Compliance Operator deployment
	coUnavailable = "unavailable"
```

Reply:
yeah, I wanted to make it descriptive as the audience will be mainly the PMs and it is easy to confuse `unknown` with `unavailable`. Let me see if I can improve it further.

Reply:
@parametalol PTAL again. 
Moreover, I tested that on a cluster and it works - see PR description.